packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5489,25 +5492,35 @@ class WidgetToRenderBoxAdapter extends LeafRenderObjectWidget {
 ///
 /// See also:
 ///
-///  * [MouseTracker] an object that tracks mouse locations in the [GestureBinding].
-class Listener extends SingleChildRenderObjectWidget {
+///  * [MouseRegion], a similar widget that tracks mouse locations even when
+///    no button is pressed.
+class Listener extends StatelessWidget {
   /// Creates a widget that forwards point events to callbacks.
   ///
   /// The [behavior] argument defaults to [HitTestBehavior.deferToChild].
   const Listener({
     Key key,
     this.onPointerDown,
     this.onPointerMove,
-    this.onPointerEnter,
-    this.onPointerExit,
-    this.onPointerHover,
+    // We have to ignore the lint rule here in order to use deprecated
+    // parameters and keep backward compatibility.
+    // TODO(tongmu): After it goes stable, remove these 3 parameters from Listener
+    // and Listener should no longer need an intermediate class _PointerListener.
+    // https://github.com/flutter/flutter/issues/36085
```

Comment:
lets add this to https://github.com/flutter/flutter/wiki/Pending-deprecations as well